src/Microsoft.Windows.CsWin32/GeneratorOptions.cs

```diff
@@ -106,5 +106,11 @@ public record FriendlyOverloadOptions
         /// </summary>
         /// <value>The default value is <see langword="true" />.</value>
         public bool Enabled { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to generate overloads that use pointer types for parameters that are [MemorySize] annotated buffers
+        /// which normally appear as spans.
+        /// </summary>
+        public bool PointerOverloadsToo { get; set; } = false;
```

Comment:
based on the naming of other properties: consider IncludePointerOverloads or IncludePointerAsOverloads

Reply:
IncludePointerOverloads sounds good to me. Renamed.